common/djangoapps/enrollment/tests/test_views.py

```diff
@@ -1395,3 +1396,81 @@ def test_response_valid_queries(self, args):
         results = content['results']
 
         self.assertItemsEqual(results, expected_results)
+
+    def _create_multiple_users_and_enroll_in_course(self, count, course):
+        for i in range(1, count+1):
+            username = 'temp{}'.format(i)
+            student = UserFactory(
+                username=username,
+                email='{}@example.com'.format(username),
+                password='edx'
+            )
+            with freeze_time(self.CREATED_DATA):
+                data.create_course_enrollment(
+                    username,
+                    unicode(course.id),
+                    'honor',
+                    True
+                )
+
+    def _assert_page_data(self, course, count, pages):
+        page_number = 0
+        url = self.url
+        while True:
+            if url is None:
+                break
+            page_number += 1
+            query_params =  {'course_id': unicode(course.id), 'page_size': 200} if '?' not in url else {}
+            response = self.client.get(url, query_params)
+            self.assertEqual(response.status_code, status.HTTP_200_OK)
+
+            content = json.loads(response.content)
+            self.assertIn('next', content)
+            self.assertIn('previous', content)
+
+            if page_number == 1:
+                self.assertIsNone(content['previous'])
+            else:
+                self.assertIsNotNone(content['previous'])
+
+            if page_number < pages:
+                self.assertIsNotNone(content['next'])
+            elif page_number == pages:
+                self.assertIsNone(content['next'])
+
```

Comment:
The above checks seem to be testing the `CursorPagination` functionality instead of the API. Can I remove it @pomegranited?

Reply:
I think this whole test is testing the pagination class actually..  And since that's built in stuff, I don't think you even need to add these tests at all?

Reply:
@pomegranited, what about the lines below which check the result data in each page against the expected results? Not sure if it is needed or not.

Reply:
I think even that is the pagination class's business, since it's the one making those decisions.  So I think the whole test can go.